pipelines/longlive/pipeline.py

```diff
@@ -112,9 +118,29 @@ def __call__(
         denoising_step_list: list[int] = None,
         manage_cache: bool = True,
     ):
-        self.prepare(
-            prompts=prompts,
-            denoising_step_list=denoising_step_list,
-            manage_cache=manage_cache,
-        )
+        # Note: prepare() was already called by frame_processor before __call__
```

Comment:
Noting a bug here that might be related to changes around this area.

https://github.com/user-attachments/assets/4ee84996-864c-4771-b7e5-5d6d85b8d155

In this video, I am using longlive and I:

1. Start the stream.
2. Disable Manage Cache
3. Click the Reset Cache button

And the video starts looping.